### ISSUE TITLE
Margin trading

### ISSUE THREAD

**Issue by davydof** [Ashlar/binance-api-node#195]
@balthazar 

Hey, Binance already supports margin trades via sockets and REST API

Do you have the plans to improve them by WS?
Thanks

```js
import Binance from 'binance-api-node'

const client = Binance({
  apiKey: 'xxx',
  apiSecret: 'xxx'
})

client.ws.user(msg => {
  // here I want to get the margin trades but now I can't
  console.log(msg)
})
```

**update:**
as I understood we should add something like:
```js
startMarginDataStream: () => privCall('/sapi/v1/userDataStream', null, 'POST', true),
stopMarginDataStream: listenKey => privCall('/sapi/v1/userDataStream', {listenKey}, 'DELETE', true),
```

here
https://github.com/Ashlar/binance-api-node/blob/7fb59cd35d3765a7fd7faf1ac8cf49a5b941f47e/src/http-client.js#L269

and after exec `startMarginDataStream`

**Comment by balthazar** [comment]
Closed per #201, will release shortly

**Comment by jsappme** [comment]
Does this pkg support margin trading?
